src/main/java/com/unzer/payment/communication/json/JsonCancel.java

```diff
@@ -103,27 +103,37 @@ public void setDate(Date date) {
         this.date = date;
     }
 
-    public Boolean isSuccess() {
+    public Boolean getSuccess() {
         return isSuccess;
     }
 
-    public void setIsSuccess(Boolean isSuccess) {
+    public void setSuccess(Boolean isSuccess) {
         this.isSuccess = isSuccess;
     }
 
-    public Boolean isPending() {
+    public Boolean getPending() {
         return isPending;
     }
 
-    public void setIsPending(Boolean isPending) {
+    public void setPending(Boolean isPending) {
         this.isPending = isPending;
     }
 
-    public Boolean isError() {
+    @Override
+    public Boolean getResumed() {
+        return null;
+    }
+
+    @Override
+    public void setResumed(Boolean value) {
+        // Do nothing
+    }
```

Comment:
I think you can use the normal setter and getter here since papi response actually also contains "isResumed" for other transaction types than chg/auth.